scripts/user-actions/search.js

```diff
@@ -3,15 +3,16 @@ console.log('--- loading: search.js');
 
 const searchHandler = () => {
   debugger;
-  const userInput = prompt('');
-  if (_) {
+  const userInput = prompt('Enter a search query: letter, syllable or word');
+  if (userInput === null) {
     return;
   }
 
   const includesSubstring = (entry) => {
-    return _;
+    return entry === userInput;
   };
-  const foundWords = words._(_);
+  const foundWord = words.filter(includesSubstring);
```

Comment:
I think you should use `includes()` method, Can you try this;

My solution;
```
return includes(entry);
};
const foundWord = words.filter(includesSubstring(userInput));

Reply:
Oguz, I tried to fix, the .includes - methods works, Thank you! Now the code is:
``` js
  const includesSubstring = (entry) => {
    return entry.includes(userInput);
  };
  const foundWord = words.filter(includesSubstring);
  displayHandler(`search results for "${userInput}": \n ${foundWord}`, []);
``` 

it works, but in the end of new array there is colon. May be you know how to remove it?
PR is updated